pkg/chains/storage/oci/oci_test.go

```diff
@@ -63,7 +63,7 @@ func TestBackend_StorePayload(t *testing.T) {
 					PayloadFormat: "in-toto",
 				},
 			},
-			wantErr: true,
+			wantErr: false,
```

Comment:
Maybe you could add a new assertion here to cover the new behavior. I'm not sure what, but maybe that assert that the log entry was seen, or the function returned nil like it should have.

Reply:
By setting `wantErr` to `false`, the assertion on line 78 will check that the return value is `nil`. I'm not sure if looking back in the logs is needed. I don't see this currently being done in the code base.

Reply:
Ack, sounds good.